### ISSUE TITLE
'taste receptor cell' is loosely defined but tightly classified

### ISSUE THREAD

**Issue by gouttegd** [obophenotype/cell-ontology#1961]
(This is another instance of the same type of problem highlighted in #1960.)

[Taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) is logically defined as a [neuron](http://purl.obolibrary.org/obo/CL_0000540) that is [capable of](http://purl.obolibrary.org/obo/RO_0002215) some [detection of chemical stimulus involved in sensory perception of taste](http://purl.obolibrary.org/obo/GO_0050912). That definition is not taxon-restricted and will therefore catch any such neuron in any species. This includes, for example, the gustatory receptor neurons of _Drosophila melanogaster_, which are similarly defined by reference to GO:0050912.

But [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) is _also_ classified as being [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [stratum basale of epidermis](http://purl.obolibrary.org/obo/UBERON_0002025), which is ultimately (through a chain of relationships going back to Uberon’s [neural crest](http://purl.obolibrary.org/obo/UBERON_0002342)) taxon-restricted to chordates.

This causes ~200 gustatory receptors neurons from FBbt to be unsatisfiable when merging Uberon, CL, and FBbt.

As in #1960, we should decide whether [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) should be taxon-neutral, in which case it should not be linked to the epidermis (beyond chordates, taste receptor cells may be found elsewhere than in the epidermis), or whether it is really intended to be chordate-specific, in which case the logical definition should be tightened.

I favour the first solution, for consistency with the neighbouring terms [olfactory epithelial cell](http://purl.obolibrary.org/obo/CL_0002167) and [auditory epithelial cell](http://purl.obolibrary.org/obo/CL_0002491), which are defined similarly to [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) but do not have an additional, taxon-restricted classification.

Of note, if we opt for the first solution of making [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) a true taxon-neutral term, its _text definition_ should also be amended, as it currently says that those cells are “mainly found in the epithelium of the tongue”, which is very much _not_ taxon-neutral.

**Comment by dosumis** [comment]
@aleixpuigb  - for cases like this, I think it will be worth having the vertebrate specific term with correct part relationships too (maybe "taste receptor cell of tongue" ?

**Comment by ybradford** [comment]
@dosumis as I note in #1963  zebrafish, a vertebrate, has taste receptors on more structures than just the tongue. https://rdcu.be/dbbA9 'The taste buds allow fish to discriminate between palatable and noxious food items, and are found throughout the oropharyngeal epithelium, on the branchial arches, and on the head and other regions of the body in some species[7](https://www.nature.com/articles/s41598-017-14042-3#ref-CR7)

**Comment by gouttegd** [comment]
@ybradford The amended definition still says “*mainly* found in the tongue”, so strictly speaking the term remains applicable to non-tongue receptors in zebrafish. :) Of course if we create a _taste receptor cell of tongue_ subclass (as suggested above), then all mention of the tongue can be removed from the parental [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209), which will make it even clearer that the term applies to any taste receptor in any species regardless of its location.

**Comment by ybradford** [comment]
@gouttegd Yes, I see what you mean. It seems that #1964 creates new term taste receptor of the tongue, so tongue can be removed from the taste receptor cell term. Thanks!

**Comment by aleixpuigb** [comment]
'taste receptor of tongue' has been created in #1966 and the second sentence (tongue specific) of the text definition of 'taste receptor cell' has been removed.
